packages/openpmd-api/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: openpmd-api
+  version: 0.14.4
+
+source:
+  url: https://github.com/openPMD/openPMD-api/archive/0.14.4.tar.gz
+  sha256: 42b7bcd043e772d63f0fe0e5e70da411f001db10096d5b8be797ffc88e786379
+  patches:
+    - 1199.patch
+
+requirements:
+  run:
+    - numpy
+    - libhdf5
+    - zlib
+
+test:
+  imports:
+    - openpmd_api
+
+build:
+  script: |
+    pip install -U pipx pybind11[global]
+    python -m pipx install cmake
+    export PATH="$(python3 -c 'import site; print(site.USER_BASE)')/bin:${PATH}"
+    pip install --upgrade pybind11[global] -t $PYODIDE_ROOT/packages/.artifacts/
+
+    export CMAKE_PREFIX_PATH=$PYODIDE_ROOT/packages/libhdf5/build/libhdf5-1.12.1/build/hdf5:$PYODIDE_ROOT/packages/zlib/build/zlib-1.2.11:${CMAKE_PREFIX_PATH}
```

Comment:
Now the last thing left is to actually find the _install_ root path of HDF5.
It looks like we cannot pick HDF5 up from its build directory directly.